app/models/user.rb

```diff
@@ -18,7 +18,7 @@
 class User < ApplicationRecord
   include AASM
 
-  DFE_EMAIL_PATTERN = '@(digital.){0,1}education.gov.uk$'.freeze
+  DFE_EMAIL_PATTERN = '@(digital\.){0,1}education.gov.uk$'.freeze
```

Comment:
good spot

Reply:
Just realised I forgot about the other two `.`. Updating now.